core/api/extrinsic/extrinsic_api_service.cpp

```diff
@@ -9,8 +9,10 @@
 #include "api/transport/impl/listener_impl.hpp"
 
 namespace kagome::api {
-  ExtrinsicApiService::ExtrinsicApiService(std::shared_ptr<Listener> listener,
-                                           std::shared_ptr<ExtrinsicApi> api)
+  ExtrinsicApiService::ExtrinsicApiService(
+      std::shared_ptr<Listener> listener,
+      std::shared_ptr<ExtrinsicApi>
+          api)  // NOLINT(performance-unnecessary-value-param)
       : ApiService(std::move(listener),
                    std::make_shared<ExtrinsicJRPCProcessor>(std::move(api))) {}
```

Comment:
why `ExtrinsicJRPCProcessor` is not passed in constructor? it should not be created here